dashboard/legacy/middleware/helpers/profanity_privacy_helper.rb

```diff
@@ -16,18 +16,6 @@ def initialize(flagged_text)
 # Check whether it is defined—helpers can be double-included during test running.
 BLOCKLY_SOURCE_FILENAME = 'main.json'.freeze unless defined? BLOCKLY_SOURCE_FILENAME
 
-def profanity_privacy_violation?(filename, body)
-  return false unless filename == BLOCKLY_SOURCE_FILENAME
-  share_failure = share_failure_from_body body, request.locale
-  !!share_failure
-end
-
-def channel_policy_violation?(channel_id)
```

Comment:
Removed since its only use was in endpoint `get %r{/v3/channels/([^/]+)/privacy-profanity}` that is being deprecated: